src/ddl/migrations/fluview_clinical_v0.1.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE fluview_clinical ADD CONSTRAINT `release_date` UNIQUE(`release_date`, `epiweek`, `region`);
```

Comment:
is it useful to put everything in one file?

```suggestion
DELETE FROM fluview_clinical WHERE id in ( SELECT dup.id FROM ( SELECT  * FROM ( SELECT fc.id, fc.release_date, fc.issue, fc.epiweek, fc.region, fc.lag, fc.total_specimens, fc.total_a, fc.total_b, fc.percent_positive, fc.percent_a, fc.percent_b, ROW_NUMBER() OVER ( PARTITION BY fc.release_date,fc.issue,fc.epiweek,fc.region,fc.lag,fc.total_specimens,fc.total_a,fc.total_b,fc.percent_positive,fc.percent_a,fc.percent_b ORDER BY fc.release_date ) as row_num FROM epidata.fluview_clinical fc ORDER BY fc.release_date DESC ) s WHERE s.row_num > 1 ) dup );
ALTER TABLE fluview_clinical ADD CONSTRAINT `release_date` UNIQUE(`release_date`, `epiweek`, `region`);
```

Reply:
yeah, we may do like that. I just thought that we need to create migration files separately.

Reply:
if nothing else, that would save a copy of the command used to version control, should we need to reference it later 👍 

also, will MySQL complain that we are trying to create an INDEX (aka KEY) with a name that already exists?  or will it simply overwrite the old one, or something else entirely?

Reply:
@melange396 I tried it locally and it didn't complain about that. Just said that I need to remove the old one or rename this (as you mentioned somewhere above)

Reply:
yeah, a quick removal of the old one before creating the new one should do the trick!